### ISSUE TITLE
[Alpha 2] Every game boot loops to Dreamcast main menu after attempted boot, with "Please insert disc" error

### ISSUE THREAD

**Issue by Drakonas** [mrneo240/GDMENUCardManager#6]
**Describe the bug**
Games booted on alpha 1, but no games will boot with alpha 2. It just loops to the Dreamcast main menu when attempting to boot any game.

**To Reproduce**
1. Install alpha 2 over the alpha 1 install on the SD card, or delete the 01 folder and copy the GDEmu GDI files over again, then run alpha 2 GDMenuCardManager install.
2. Start Dreamcast with GDEmu installed and SD card inserted with alpha 2 installed. You should see the Dreamcast logo and then OpenMenu 6.24.21..
3. Start a game with 'A'.

**Expected behavior**
Game should boot, with or without the logo.

**Actual behavior**
Dreamcast logo shows again, then the main Dreamcast menu is loaded. Attempting to load "Game" shows the "Please insert a disc" error. Rebooting the Dreamcast goes back into GDEmu menu, and the process starts over.

**Desktop (please complete the following information):**
 - OS: Windows 10 21H1
 - Browser: Opera
 - Browser Version: 77.0.4054.90

**Dreamcast (please complete the following information):**
 - Version: V1 USA

**Additional context**
I can confirm that the same games booted fine on alpha 1. I am using a SanDisk A2 512GB Micro SD card.

**Comment by Drakonas** [comment]
I should note that I had initially added games with GDMenuCardManager alpha 1. Since the renaming functionality was broken, I manually renamed the files myself in the name.txt files myself using Sublime Text 4, making sure that the name doesn't go past 40 characters, which seems to be the limit.

Afterwards, I had actually opened GDEmu's official SD manager to tick the "VGA" flag for all the games. I don't know if this may be the cause of my trouble.

The next time I used GDMenuCardManager after that was Alpha 2.
